cargo-generate/configure_files.rhai

```diff
@@ -1,4 +1,14 @@
-file::delete("Cargo.toml");
-file::rename("cargo-generate/Cargo.toml", "Cargo.toml");
+switch variable::get("yew_version"){
+  "stable" => {
+    file::rename("cargo-generate/Cargo.lock", "Cargo.lock");
+    file::rename("cargo-generate/Cargo.toml", "Cargo.toml");
+  }
+  "next" => {
+    file::rename("cargo-generate/Cargo.lock.next", "Cargo.lock");
+    file::rename("cargo-generate/Cargo.toml.next", "Cargo.toml");
+  }
+}
+file::rename("cargo-generate/_README.md", "README.md");
```

Comment:
cargo-generate actually strips `.liquid` suffix, so writing the full name like `cargo-generate/Cargo.lock.liquid` would cause a file not found error